components/molecule/rating/src/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import Star from './components/Star'
+
+const SIZES = {
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  LARGE: 'large'
+}
+
+const BASE_CLASS = `sui-MoleculeRating`
+const CLASS_CONTAINER_STARS = `${BASE_CLASS}-containerStars`
+const CLASS_LABEL = `${BASE_CLASS}-label`
+const CLASS_LINK = `${BASE_CLASS}--withLink`
+
+const MoleculeRating = ({
+  numStars = 5,
+  value,
+  size = SIZES.SMALL,
+  label,
+  href,
+  target,
+  link,
```

Comment:
what's the point of this boolean prop? I guess if either href or target is defined you must create a link, don't you? Maybe you can add an object prop for link that contains {href, target} and if defined, then use link, otherwise label.

Reply:
Sí, se puede mejorar el código para evitar la prop `link`. Le doy una vuelta al código